Description: Prevent building uImage with missing mkimage
 On some powerpc platforms, CONFIG_DEFAULT_UIMAGE is selected automatically,
 which leads to uImage being built automatically with mkimage. This tool is not
 available in linux's build-dependencies, and the file is not strictly
 necessary, so we are omitting this step in the build process, Debian-specific.
Author: Roland Stigge <stigge@antcom.de>
Bug-Debian: https://bugs.debian.org/708094
Forwarded: not-needed

--- a/arch/powerpc/boot/Makefile
+++ b/arch/powerpc/boot/Makefile
@@ -221,7 +221,6 @@ image-$(CONFIG_PPC_CHRP)		+= zImage.chrp
 image-$(CONFIG_PPC_EFIKA)		+= zImage.chrp
 image-$(CONFIG_PPC_PMAC)		+= zImage.pmac
 image-$(CONFIG_PPC_HOLLY)		+= dtbImage.holly
-image-$(CONFIG_DEFAULT_UIMAGE)		+= uImage
 image-$(CONFIG_EPAPR_BOOT)		+= zImage.epapr
 
 #
@@ -280,23 +279,6 @@ image-$(CONFIG_MPC834x_MDS)		+= cuImage.
 image-$(CONFIG_MPC836x_MDS)		+= cuImage.mpc836x_mds
 image-$(CONFIG_ASP834x)			+= dtbImage.asp834x-redboot
 
-# Board ports in arch/powerpc/platform/85xx/Kconfig
-image-$(CONFIG_MPC8540_ADS)		+= cuImage.mpc8540ads
-image-$(CONFIG_MPC8560_ADS)		+= cuImage.mpc8560ads
-image-$(CONFIG_MPC85xx_CDS)		+= cuImage.mpc8541cds \
-					   cuImage.mpc8548cds_32b \
-					   cuImage.mpc8555cds
-image-$(CONFIG_MPC85xx_MDS)		+= cuImage.mpc8568mds
-image-$(CONFIG_MPC85xx_DS)		+= cuImage.mpc8544ds \
-					   cuImage.mpc8572ds
-image-$(CONFIG_TQM8540)			+= cuImage.tqm8540
-image-$(CONFIG_TQM8541)			+= cuImage.tqm8541
-image-$(CONFIG_TQM8548)			+= cuImage.tqm8548
-image-$(CONFIG_TQM8555)			+= cuImage.tqm8555
-image-$(CONFIG_TQM8560)			+= cuImage.tqm8560
-image-$(CONFIG_SBC8548)			+= cuImage.sbc8548
-image-$(CONFIG_KSI8560)			+= cuImage.ksi8560
-
 # Board ports in arch/powerpc/platform/86xx/Kconfig
 image-$(CONFIG_MVME7100)                += dtbImage.mvme7100
 
